src/main/java/com/hotels/service/tracing/zipkintohaystack/metrics/MetersProvider.java

```diff
@@ -34,14 +33,22 @@ public MetersProvider(MeterRegistry meterRegistry) {
     }
 
     public Counter getSpansCounter(String protocol, String transport) {
-        return Counter.builder("counter.pitchfork.spans")
+        return Counter.builder("counter.pitchfork.spans.ingress")
                 .tags("transport", transport, "protocol", protocol)
                 .register(meterRegistry);
     }
 
+    // TODO: wouldnt it be nice if we could merge these counters with the above? ^
     public Counter getInvalidSpansCounter() {
```

Comment:
They need to have the same set of tags otherwise micrometer will throw an exception. Added a todo to fix this later.